### ISSUE TITLE
sqlalchemy fails with 3.8.0b2

### ISSUE THREAD

**Issue by lazka** [python/pyperformance#62]
```python
INFO:root:Running `C:\Python38\venv\cpython3.8-2459c2ec2c3d\Scripts\python.exe -u C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_sqlalchemy_imperative.py --output C:\Users\xy\AppData\Local\Temp\tmpidjg6d0q`
Traceback (most recent call last):
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\pyperformance\benchmarks\bm_sqlalchemy_imperative.py", line 4, in <module>
    from sqlalchemy import Column, ForeignKey, Integer, String, Table, MetaData
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\sqlalchemy\__init__.py", line 8, in <module>
    from . import util as _util  # noqa
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\sqlalchemy\util\__init__.py", line 14, in <module>
    from ._collections import coerce_generator_arg  # noqa
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\sqlalchemy\util\_collections.py", line 16, in <module>
    from .compat import binary_types
  File "C:\Python38\venv\cpython3.8-2459c2ec2c3d\lib\site-packages\sqlalchemy\util\compat.py", line 331, in <module>
    time_func = time.clock
AttributeError: module 'time' has no attribute 'clock'
ERROR: Benchmark sqlalchemy_imperative failed: Benchmark died
```

**Comment by lazka** [comment]
Fixed by https://github.com/sqlalchemy/sqlalchemy/issues/4731

**Comment by vstinner** [comment]
Fixed in just releaesd pyperformance 0.9.1 by commit d631ab36e7a07f8a7f17591a922df39818cb724e. Thanks for the bug report and thanks for reporting the issue to sqlalchemy!
